app/views/facilities/edit.html.erb

```diff
@@ -1,25 +1,34 @@
 <%=form_for @facility do |f|%>
-    
-  
-    <div>
-        <%=f.text_field  :name ,placeholder:"Name of Facility"%> 
-    </div>
-    
-    <div>
-        <%=f.text_field :city ,placeholder:"City"%> 
-    </div>
-    <div>
-        <%=f.text_field :state ,placeholder:"State"%> 
-    </div>
-    <div>
-        <%=f.text_field :zip ,placeholder:"ZIP Code"%> 
-    </div>
-    <div>
-        <%=f.text_field :specialty ,placeholder:"Speciality of Facility"%> 
-    </div>
-    
-     <div>
-        <%=f.submit "Post"%>
-    </div>
 
-<%end%>
\ No newline at end of file
+  <div>
+    <%= f.text_field  :name, placeholder: "Name of Facility" %>
+    <%= f.label :facility, 'Facility' %>
+  </div>
+  <div>
+    <%= f.text_field :street, placeholder: "Street Adress" %>
+    <%= f.label :street_address, 'Street Adress' %>
```

Comment:
The label tag needs to exactly match the parameter name; this is a case where using the extra string parameter is helpful. (Also, misspelled 'address' on both lines 8 and 9).

```suggestion
    <%= f.label :street, 'Street Address' %>
```